packages/hyperion-autologging/test/ALInteractableDOMElement.test.ts

```diff
@@ -108,4 +108,28 @@ describe("Text various element text options", () => {
 
     dom.cleanup();
   });
+
+  test("Detect data-*able is set correctly", () => {
+    const dom = DomFragment.html(`
+      <div id="attribute"></div>
+      <div id="addEventListener"></div>
+    `);
+
+    ALInteractableDOMElement.trackInteractable("click");
+
+    let node = document.getElementById("attribute");
+    expect(node).not.toBeNull();
+    if (node) {
+      node.onclick = () => { };
+      expect(node.getAttribute("data-clickable")).toBe("1");
```

Comment:
minor but could also include a test setting `node.onclick = null` and see if the attribute gets removed.